tests/lib/tools/os.query

```diff
@@ -64,7 +64,8 @@ is_debian() {
 }
 
 is_debian_sid() {
-    grep -qFx 'ID=debian' /etc/os-release && grep -qx 'PRETTY_NAME=".*/sid"' /etc/os-release
+    #TODO: remove check for bullseye codename once the os-release file is updated with sid pretty name again
+    grep -qFx 'ID=debian' /etc/os-release && ( grep -qx 'PRETTY_NAME=".*/sid"' /etc/os-release || grep -qFx  'VERSION_CODENAME=bullseye' /etc/os-release )
```

Comment:
AFAIU this will break again once base-files 11.1 lands and it's bookworm (?).

Also, the changelog https://metadata.ftp-master.debian.org/changelogs//main/b/base-files/base-files_11.1_changelog mentions it's supposed to be `bullseye/sid`, but all I can see is `Debian GNU/Linux 11` in `/etc/issue`, or just `bullseye` elsewhere. TBH I have no clue how to identify system as sid properly at this point, but that can be a garbage too. Maybe we should just look at SPREAD_SYSTEM?

```
grep -qFx 'ID=debian' /etc/os-release && echo "$SPREAD_SYSTEM" | grep -qFx debian-sid-64
```

Reply:
@bboozzoo thanks for the review, It is ok for me to use the SPREAD_SYSTEM. 
@mvo5 any suggestion?